packages/fluentui/react-northstar/src/components/Grid/Grid.tsx

```diff
@@ -39,12 +39,12 @@ class Grid extends UIComponent<WithAsProp<GridProps>> {
     ...commonPropTypes.createCommon({
       content: false,
     }),
-    columns: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    columns: PropTypes.oneOfType([PropTypes.number]),
     content: customPropTypes.every([
       customPropTypes.disallow(['children']),
       PropTypes.oneOfType([PropTypes.arrayOf(customPropTypes.nodeContent), customPropTypes.nodeContent]),
     ]),
-    rows: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    rows: PropTypes.oneOfType([PropTypes.number]),
```

Comment:
you can probably put just PropTypes.number and remove the `oneOfType`. same above.

Reply:
Definitely right, thanks :)